includes/Plugin_Context.php

```diff
@@ -98,4 +102,34 @@ public function location() {
 	public function is_single_file_plugin() {
 		return $this->path() !== $this->location();
 	}
+
+	/**
+	 * Determine the minimum supported WordPress version of the plugin.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return bool|string The minimum version supported, otherwise false.
```

Comment:
Nit-pick: A cleaner implementation would be to always return a `string`. Better to avoid multiple return types if possible.
```suggestion
	 * @return string The minimum version supported, or empty string if unknown.
```